docker-compose.yml

```
@@ -17,7 +41,21 @@ services:
       - ocr-cache:/root/.paddleocr
-    expose:
+    ports:
+      - "8000:8000"
+    healthcheck:
+      test: ["CMD", "wget", "-qO-", "http://localhost:8000/healthz"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+      start_period: 10s
+      - "8000"
+    healthcheck:
```

YAMLlint (1.37.1)

[error] duplication of key "healthcheck" in mapping

(key-duplicates)